### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by DietrichWasHere**
THE FINAL MERGE INTO MASTER!!!!!!!

DO NOT DO UNTIL OK'D

Issues to fix:
- pull in the branch that fixed the link (quick fix and force deploy) to fix a link and force dev to deploy to dev server
- conflicts
- maybe fix css in a couple of places (looks mostly good)

### CODE REVIEW

**Review by DietrichWasHere — COMMENTED** [review]
Looks good, apart from a few css hiccups and missing link

Don't know how to resolve the conflicts myself; its telling me to use the command line?

File: Howtoprepare.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        </div>
+        <h1 class="title">How To Prepare For CS?</h1>
+        <div></div>
+        <div class="top">
+            <h2 class="subtitle">There are many ways to prepare for CS</h2>
+            <p class = "generalLight">
+              &emsp;We would recommend that you check out w3 schools. <br>
+              &emsp;They have many great tutorials on coding that are<br>
+              &emsp;easy to learn and will actually be used in your classes.<br>
+              &emsp;CLick <a href = "https://www.w3schools.com"  class="generalLight">here</a> to go to w3 schools<p></p>
```
Comment (TimothyWhiteOC) [inline Howtoprepare.html]: Fix capitalization
```suggestion
              &emsp;Click <a href = "https://www.w3schools.com"  class="generalLight">here</a> to go to w3 schools<p></p>
```

File: Howtoprepare.html [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+            </p>
+        </div>
+        <div class = "bottom">
+            <h2 class ="subtitle">Learn a Programming Language</h2>
+            <p class="generalDark">
+                &emsp;If you would like a recommendation for a language to <br>
+                &emsp;learn before classes start, C++ is going to be an <br>
+                &emsp;excellet first choice. Its what you will learn in <br>
```
Comment (TimothyWhiteOC) [inline Howtoprepare.html]: Fix spelling
```suggestion
                &emsp;excellent first choice. It's what you will learn in <br>
```

File: Howtoprepare.html [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+                &emsp;Programming 1 and 2.Knowledge of this beforehand will <br>
```
Comment (TimothyWhiteOC) [inline Howtoprepare.html]: Add space
```suggestion
                &emsp;Programming 1 and 2. Knowledge of this beforehand will <br>
```

File: Howtoprepare.html [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+                &emsp;help you excell in your classes.
```
Comment (TimothyWhiteOC) [inline Howtoprepare.html]: Fix spelling
```suggestion
                &emsp;help you excel in your classes.
```

File: business.html [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+    </div>
+        <br><br>
+        <div class="top">
+            <h2 class="subtitle">What is it?</h2>
+            <p class="generalLight">
+                &emsp;Computer Science with Business is a degree that combines all<br>
+                &emsp;of the theoretical basics of Computer Science while including<br>
+                &emsp;a the basic foundations of business. This allows for a technical<br>
+                &emsp;degree with strong foundations in interpersonal business skills</p>
```
Comment (TimothyWhiteOC) [inline business.html]: Add period
```suggestion
                &emsp;degree with strong foundations in interpersonal business skills.</p>
```

File: business.html [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+        </div>
+        <br><br>
+        <div class="bottom">
+            <h2 class="subtitle">Job Opportunities:</h2>
+            <p class="generalDark">
+                &emsp;Computer Science with business equips alumni to excell in large<br>
```
Comment (TimothyWhiteOC) [inline business.html]: Fix spelling
```suggestion
                &emsp;Computer Science with business equips alumni to excel in large<br>
```

File: business.html [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+                &emsp;firms, start-up companies, and can ultimatley lead to a senior<br>
```
Comment (TimothyWhiteOC) [inline business.html]: Fix spelling
```suggestion
                &emsp;firms, start-up companies, and can ultimately lead to a senior<br>
```

File: business.html [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+                &emsp;managment position. A popular position that encompases both<br>
```
Comment (TimothyWhiteOC) [inline business.html]: Fix spelling
```suggestion
                &emsp;management position. A popular position that encompasses both<br>
```

File: business.html [hunk 9]
```diff
@@ -0,0 +1,70 @@
...
+                &emsp;focuses is called a Scrum Master</p>
```
Comment (TimothyWhiteOC) [inline business.html]: Add period
```suggestion
                &emsp;focuses is called a Scrum Master.</p>
```

File: careers.html [hunk 10]
```diff
@@ -0,0 +1,81 @@
...
+                &emsp;That is more than double the national average wage<br>
+                &emsp;($47,060)</p>
+        </div>
+        <br><br>
+        <div class="bottom">
+            <h2 class="subtitle">Potential Growth:</h2>
+            <p class="generalDark">
+                &emsp;Jobs involved with Computer Science are expected to grow by a whopping <br>
+                &emsp;16% by 2029 adding in more than 600,000 new jobs into the market by<br>
+                &emsp; then. This results in lower difficulty entering the job market right <br>
```
Comment (TimothyWhiteOC) [inline careers.html]: Delete extra space
```suggestion
                &emsp;then. This results in lower difficulty entering the job market right <br>
```

File: gamingAndAnim.html [hunk 11]
```diff
@@ -0,0 +1,60 @@
...
+        <script src="dropdown-menu.js"></script>          
+        <h1 class="title">Computer Science<br>with Gaming and Animation</h1>
+        <div class = "section">
+            <h2 class = "subtitle">Student Experience</h2>
+            <p class ="generalDark">
+                This set of entertainment-focused specializations brings art, storytelling, and interaction to the forefront.
+                Without these programmers, animators and game development teams would be unable to realize their vision. Furthermore,
+                these programmers may be able to play a role in making some design decisions, depending on their integration into the team.
+                Due to the artistic nature of game development, more versatility is embedded into a Computer Science with
+                Gaming and Anomation degree than a traditional CS degree, if only to better enable communication and team work within the industry.
```
Comment (TimothyWhiteOC) [inline gamingAndAnim.html]: Fix spelling
```suggestion
                Gaming and Animation degree than a traditional CS degree, if only to better enable communication and teamwork within the industry.
```

File: index.html [hunk 12]
```diff
@@ -1,7 +1,115 @@
...
+    <br>
+    <div class = "section">
+      <br>
+      <a href="index.html" class = "subtitle">The Art of Programming</a>
+      <p class ="generalDark">
+        &emsp;Computer science is the art of communication through logic. It is <br>
+        &emsp;about understanding how to make a logical machine do what you desire,<br>
+        &emsp;and often times involves enabling the machine to communicate with an <br>
+        &emsp;end user. This is accomplished through the use of various programming <br>
+        &emsp;languages, and using a corresponding compiler to translate the written<br>
```
Comment (TimothyWhiteOC) [inline index.html]: Move comma
```suggestion
        &emsp;languages, and using a corresponding compiler to translate the written,<br>
```

File: index.html [hunk 13]
```diff
@@ -1,7 +1,115 @@
...
+        &emsp;, human readable "code" into electrical pulses. These pulses are read<br>
```
Comment (TimothyWhiteOC) [inline index.html]: Move comma
```suggestion
        &emsp;human readable "code" into electrical pulses. These pulses are read<br>
```

File: index.html [hunk 14]
```diff
@@ -1,7 +1,115 @@
...
+        &emsp;by humans as a sequence of ones and zeros, referred to as binary. <br>
+        &emsp;These electrical signals can be transferred, recorded, interpreted, <br>
+        &emsp;edited, and operated on by cicrutry and mechanisms in the computer. <br>
```
Comment (TimothyWhiteOC) [inline index.html]: Fix spelling
```suggestion
        &emsp;edited, and operated on by circuitry and mechanisms in the computer. <br>
```

File: index.html [hunk 15]
```diff
@@ -1,7 +1,115 @@
...
+        &emsp;This includes handling data (numbers and letters) or reading and <br>
+        &emsp;ennacting instructions and operations from a computer's instruction set.<br>
```
Comment (TimothyWhiteOC) [inline index.html]: Fix spelling
```suggestion
        &emsp;enacting instructions and operations from a computer's instruction set.<br>
```

File: index.html [hunk 16]
```diff
@@ -1,7 +1,115 @@
...
+        &emsp;These instructions are the opposite binary counterpart of the code, <br>
+        &emsp;and sequences of them make up the logic that drives the computer and its<br> 
+        &emsp;applications. These instructions thereby determine how end users <br>
+        &emsp;interact with there machines.<br>
```
Comment (TimothyWhiteOC) [inline index.html]: Fix spelling
```suggestion
        &emsp;interact with their machines.<br>
```

File: index.html [hunk 17]
```diff
@@ -1,7 +1,115 @@
...
+    </div>
+
+    <br><br><br><br><br>
+    <div class = "top">
+        <a href="Whystudy.html" class = "subtitle">Why Study Computer Science?</a>
+        <p class="generalLight">
+          &emsp;Computer science is a rising industry that constantly requires a new<br> 
+          &emsp;influx of talent. Between its wealth of career paths (including <br>
+          &emsp;application development, cybersecurity, and game development), its <br>
+          &emsp;pay, and its intrinsic interest. Computer science may well be the<br>
```
Comment (TimothyWhiteOC) [inline index.html]: Fix incomplete sentence
```suggestion
          &emsp;pay, and its intrinsic interest, computer science may well be the<br>
```

File: index.html [hunk 18]
```diff
@@ -1,7 +1,115 @@
...
+        </p>
+    </div>
+    <br><br>
+    <div class = "bottom">
+      <a href="what-its-like.html" class = "subtitle">What's It Like to Study CS?</a>
+      <p class = "generalDark">
+        &emsp;Computer Science is a relatively rigorous course of study. You'll <br>
+        &emsp;need to learn to program in various laguages, and to understand <br>
+        &emsp;the core logic that transcends these languages. You'll also learn<br>
+        &emsp; different tools tied to your specialization. Another key aspect <br>
```
Comment (TimothyWhiteOC) [inline index.html]: Remove extra space
```suggestion
        &emsp;different tools tied to your specialization. Another key aspect <br>
```

File: what-its-like.html [hunk 19]
```diff
@@ -0,0 +1,82 @@
...
+                &emsp;information, you will be doing things like learning different<br>
+                &emsp;programming languages, studying algorithms, using logic to <br>
+                &emsp;solve issues, and working in teams.</p>
+        </div>
+        <br><br>
+        <div class="bottom">
+            <h2 class="subtitle">Is it hard?</h2>
+            <p class="generalDark">
+                &emsp;One of the most common questions from highschool students is<br>
+                &emsp;"Is Computer Science Hard?" Unfortunatley, this major has the highest<br>
```
Comment (TimothyWhiteOC) [inline what-its-like.html]: Fix spelling
```suggestion
                &emsp;"Is Computer Science Hard?" Unfortunately, this major has the highest<br>
```

File: what-its-like.html [hunk 20]
```diff
@@ -0,0 +1,82 @@
...
+                &emsp;drop out rate out of all majors. If you are logically inclined and<br>
+                &emsp;enjoy a bit of a challenge, computer science might be for you</p>
```
Comment (TimothyWhiteOC) [inline what-its-like.html]: Add period
```suggestion
                &emsp;enjoy a bit of a challenge, computer science might be for you.</p>
```

**Review by TimothyWhiteOC — CHANGES_REQUESTED** [review]
Several spelling/grammar issues

**Review by TimothyWhiteOC — APPROVED** [review]
Ready to merge

**Review by Braden-Thompson — APPROVED** [review]

**Review by EddieAGomezSchieber — APPROVED** [review]
Looks good

**Pull request merged**